tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
Well, symbols make it certain there won't be unintended name collisions. But sure, we can go with some weird name like ``baseToolsInit`` if you are that against Symbols.